dynawo/sources/Modeler/DataInterface/DYNStaticVarCompensatorInterface.h

```diff
@@ -161,6 +161,12 @@ class StaticVarCompensatorInterface : public ComponentInterface {
    */
   virtual double getB0() const = 0;
 
+  /**
+   * @brief Getter for the active power of static var compensator in Mvar (Receptor convention)
+   * @return The active power of the static var compensator in Mvar (Receptor convention)
+   */
+  virtual double getP() = 0;
```

Comment:
const?

Reply:
All methods getP and getQ are not const in all the Interface and InterfaceIIDM classes, I think it is because it tests if the component is connected and return 0 if it is not. What do you think?

Reply:
that's correct!